kafka/conn.py

```diff
@@ -70,6 +71,9 @@ class BrokerConnection(object):
         'ssl_password': None,
         'api_version': (0, 8, 2),  # default to most restrictive
         'state_change_callback': lambda conn: True,
+        'sasl_mechanism': None,
+        'sasl_username': None,
+        'sasl_password': None
```

Comment:
I dont see sasl.username / sasl.password in the java kafka docs. I see some reference to plain user/pass stored in Jaas configuration files. Is that the standard method for java?

For kafka-python, should we add plain_ : `sasl_plain_username` etc?